doc/Roadmap.md

```diff
@@ -51,24 +71,6 @@ Please contact us and tell us about what features and improvements do you see as
 * Stories?
 * Contributions?
 
-## Current state of the MongooseIM platform
-
-This is not the MongooseIM XMPP/REST messaging server, all these components are part of the MongooseIM platform.
-
-New components:
-
-* [MongoosePush](https://github.com/esl/MongoosePush): open source Push notifications server in Elixir
-* [MongooseICE](https://github.com/esl/Fennec): open source ICE/STUN/TURN server in Elixir
-* [Mangosta Android](https://github.com/esl/mangosta-android): chat and social client, sample app
-* [Mangosta iOS](https://github.com/esl/mangosta-ios): chat and social client, sample app
-* Mangosta Web? Mangosta Laptop (Windows, macOS, Linux)? Strophe.js? Stanza.io? React Native?
-
-Existing components:
+## Influence the vision and roadmap
 
-* [MongooseIM](https://github.com/esl/MongooseIM): MongooseIM XMPP/REST messaging cluster
-* [exml](https://github.com/esl/exml): XML library
-* [escalus](https://github.com/esl/escalus): Erlang XMPP client
-* [amoc](https://github.com/esl/amoc): A Murder Of Crows, load injector
-* [Tide/CLT](http://tide.erlang-solutions.com/): Continuous Load Testing platform
-* [Smack](https://github.com/igniterealtime/Smack): open source, third party library for Android
-* [XMPPFramework](https://github.com/robbiehanson/XMPPFramework): open source, third party library for iOS
+You have the opportunity to influence the future of the MongooseIM platform: please contact us and explain your wishes!
```

Comment:
Our roadmap is build based on community feedback, customer demand and our own ideas. We would love to invite everyone to contribute their vision of where we should go next.
No matter if you love, use or just know MongooseIM, share your passion and inspire the next steps of our journey.